Feat/bank feeds api (#852)
codatio/codat-docs: docs/bank-feeds/setup.md

    participant app as Your application 
    participant codat as Codat
    participant acctg as Accounting platform
    
    smb ->> app: Logs into application
    smb ->> app: Initiates connection to accounting software

    app ->> codat: Passes company and connection details
    app ->> codat: Initiates auth flow
    codat -->> smb: Displays auth flow
    smb -->> codat: Authorizes connection
    codat ->> acctg: Establishes connection
    
    codat ->> smb: Displays mapping options
    smb -->> codat: Confirms mapping selections
    
    loop Load bank statements
        smb ->> app: Spends money from bank account
        app ->> codat: Pushes bank transaction details
        codat ->> acctg: Creates bank transactions
        acctg ->> smb: Displays loaded bank statement ready for reconciliation
    end

```

## Configure Bank Feeds API

Once you decide to build this flow with Bank Feeds API, you need to configure Codat accordingly. Let's go through these requirements in detail.

### Enable the product

1. Open the <a href="https://app.codat.io" target="_blank">Codat Portal</a> and sign in.
2. Click on **Settings > Organizational settings > Products**.

> 3. In the list of products, find _Bank Feeds API_ and click **Enable**. Then, follow the on-screen prompt.

Reviewer: We should follow Lending API documentation? 
- Level 2 title and before configure product
- Should be titled 'Enable Bank Feeds API'

### Manage data sources

In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate to **Settings > Integrations** and click **Manage integrations**. Next, click **Manage** next to the specific integration you want to enable and set it up to serve as a data source for the product. 

<IntegrationsList integrations={bankfeedsIntegrations}/>

> Some of these integrations may require enhanced setup specific to bank feeds. We walk you through these in our integration-specific instructions in the _Manage integrations_ section of our Bank Feeds API documentation. 

Reviewer: ```suggestion
Some of these integrations require additional setup specific to bank feeds. We walk you through these in our integration-specific instructions in the _Manage integrations_ section of our Bank Feeds API documentation. 
```

### Authorization flow

> As part of using Bank Feeds API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization flow. 

Reviewer: ```suggestion
As part of using Bank Feeds API, you will need your customers to authorize your access to their data. To do so, use [Embedded Link](https://codat-docs-git-feat-bank-feeds-api-codat.vercel.app/auth-flow/authorize-embedded-link) - our pre-built, embeddable, conversion-optimized, and white-labelled authorization flow. 
```

> We recommend you fully embed the Link auth flow in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) SDK in your front-end code. You can also choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) auth flow option to get up and running as quick as possible. 

Reviewer: We should be opinionated about what type of link they use.

> We recommend you fully embed the Link auth flow in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) SDK in your front-end code. You can also choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) auth flow option to get up and running as quick as possible. 

Reviewer: ```suggestion
```

The solution lets you tailor the authorization journey to your business needs. You can:

* [Customize Link settings](/auth-flow/customize/customize-link).
* [Set up company branding](/auth-flow/customize/branding).
* [Set up redirects](/auth-flow/customize/set-up-redirects).

### Webhooks

Codat supports a range of [webhooks](/using-the-api/webhooks/core-rules-types) to help you manage your data pipelines. In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate to **Settings > Webhooks > Rules** and click **Create new rule** to set up the following webhook and get the most out of Bank Feeds API:

> - [Push operation status has changed](/using-the-api/webhooks/core-rules-types#push-operation-status-has-changed)  

Reviewer: I've recently seen a bank feeds webhook event in our data. Is this event type missing? Bank Feed account status change
